samples/ui-kit-place-details-compact/index.ts

```diff
@@ -7,14 +7,13 @@
 // Use querySelector to select elements for interaction.
 const map = document.querySelector('gmp-map') as google.maps.MapElement;
 //@ts-ignore
-const placeDetails = document.querySelector('gmp-place-details-compact') as google.maps.PlaceDetailsElement;
 const marker = document.querySelector('gmp-advanced-marker') as any;
 
 async function initMap(): Promise<void> {
   // Request needed libraries.
   const { Map } = await google.maps.importLibrary("maps") as google.maps.MapsLibrary;
-  const { AdvancedMarkerElement, PinElement } = await google.maps.importLibrary("marker") as google.maps.MarkerLibrary;
-  const { Place } = await google.maps.importLibrary("places") as google.maps.PlacesLibrary;
+  const { AdvancedMarkerElement } = await google.maps.importLibrary("marker") as google.maps.MarkerLibrary;
+  await google.maps.importLibrary("places") as google.maps.PlacesLibrary;
```

Comment:
Seems odd to have a const for only some. Recommend removing the const variable if unused.